server/service/apple_mdm.go

```diff
@@ -394,14 +394,14 @@ func (svc *Service) ListMDMAppleConfigProfiles(ctx context.Context, teamID uint)
 
 	if teamID >= 1 {
 		// confirm that team exists
-		if _, err := svc.ds.Team(ctx, teamID); err != nil {
-			return nil, ctxerr.Wrap(ctx, err)
+		if err := svc.ds.TeamExists(ctx, teamID); err != nil {
```

Comment:
For my own understanding, why will this fix the row locking issues? (i.e. it's not a SELECT for update, so even the previous `Team` call should not have locked the row and should be fast enough except for returning the columns?)